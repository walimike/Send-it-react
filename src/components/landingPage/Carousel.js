import React, { Component } from "react";
import M from "materialize-css/dist/js/materialize.js";

class Carousel extends Component {
  componentDidMount() {
    var elems = document.querySelectorAll(".carousel");
    var config = {
      dist: 0,
      fullWidth: true,
      duration: 400,
      indicators: true,
    };
    M.Carousel.init(elems, config);
  }


  render() {
    return (
      <div className="carousel carousel-slider center">
        <div className="carousel-fixed-item center">
          <div className="fixed-item-text">
            
          </div>
        </div>
        <a className="carousel-item" href="#three!">
          <img src="http://indailytimes.com/wp-content/uploads/2019/04/Parcel-Delivery-Market.jpg" />
        </a>
      </div>
    );
  }
}

export default Carousel;
