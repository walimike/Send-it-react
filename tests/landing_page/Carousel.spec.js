import React from "react";
import { shallow } from "enzyme";

import Carousel from "../../src/components/landingPage/Carousel";


describe("Carousel", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Carousel />);
    expect(wrapper).toMatchSnapshot();
  });
});
