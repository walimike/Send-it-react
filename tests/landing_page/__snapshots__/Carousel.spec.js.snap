// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel should render without crashing 1`] = `ShallowWrapper {}`;
